### ISSUE TITLE
Ability to send event reminder

### ISSUE THREAD

**Issue by tangentfairy-zz** [CoderDojo/community-platform#568]
Request from CD Rotterdam to send an event reminder to applicants prior to event.

I think this issue https://github.com/CoderDojo/community-platform/issues/408 will cover this in the short term, but this would be a nice feature in the long term.

**Comment by DanielBrierton** [comment]
Is the desire here to have the ability to manually send out a reminder on demand, or to provide the option of an automatic reminder X hours/days before the event when creating the event?

**Comment by rosalanghammer** [comment]
Hi @DanielBrierton would be ideal to be able to turn this on when creating. If this could be default 48 hours before that would be awesome - also if it's easy it would be cool for volunteers to be able to customise this time from 48 hours to something more suited to them when turning it on.

**Comment by DanielBrierton** [comment]
Something like this?

![event_reminder_1](https://cloud.githubusercontent.com/assets/181418/14726465/6b9c8806-081a-11e6-8791-0e9579218340.png)

The input field only displays when the checkbox is checked, and becomes a required field when displayed too. I'm not too sure on the label text I've used, but I'm struggling to come up with something better. Any suggestions on improvements to the layout/text would be greatly appreciated.

**Comment by rosalanghammer** [comment]
Yes looks good to me - I would move it so it is positioned under 'Additional Options' though.

Can't think of a better label than that so roll with it :)

**Comment by rosalanghammer** [comment]
Would be super nice to have if this was custom so Dojos could add their own requirements. e.g. 'Remember to download x, y and z'

**Comment by Wardormeur** [comment]
The message of the reminder could simply contains the description of the event instead of creating another field specifically for the reminder?

**Comment by rosalanghammer** [comment]
Yes message can just be description/details of event in first instance but it would provide more value to make it more custom than that. Just want this to be noted if someone takes this issue and feels like they want to add maximum value.

**Comment by DanielBrierton** [comment]
Should use seneca-kue (see poll sms for example)
